.circleci/config.yml

```diff
@@ -29,13 +29,6 @@ workflows:
     jobs:
       - install dependencies
       - dependency vulnerability scan:
```

Comment:
D this consolidate and rename the frontend and backend vulnerability tests into a single job? It looks like this and some of the other changes in the config.yml are related to this.

Reply:
yes, they are now one job